presto-parser/src/main/java/com/facebook/presto/sql/parser/AstBuilder.java

```diff
@@ -1791,6 +1818,16 @@ private static String typeParameterToString(SqlBaseParser.TypeParameterContext t
         throw new IllegalArgumentException("Unsupported typeParameter: " + typeParameter.getText());
     }
 
+    private static List<String> getIdentifiers(List<SqlBaseParser.IdentifierContext> identifiers)
+    {
+        return identifiers.stream().map(SqlBaseParser.IdentifierContext::getText).collect(toList());
```

Comment:
`toImmutableList()` is not available in `presto-parser` module. `toList()` collector is used all across the code of this class.